latex2lambda.py

```diff
@@ -0,0 +1,41 @@
+from latex2sympy2 import latex2sympy
+from sympy import lambdify
+
+
+def latex2lambda(
+    tex: str,
+    variable_values={},
+    args=None,
+    modules=None,
+    printer=None,
+    use_imps=True,
+    dummify=False,
+):
+    if not args:
```

Comment:
PEP8, recommends doing `if args is not None`, rather than this.
https://peps.python.org/pep-0008/#programming-recommendations

Reply:
You're right!